lib/seadsa/DsaLocal.cc

```diff
@@ -519,14 +519,18 @@ class IntraBlockBuilder : public InstVisitor<IntraBlockBuilder>,
     }
   }
 
-  auto getSeaDsaAttrbFunc(const Function *fn) {   
+  typedef seadsa::Node &(seadsa::Node::*seaNodeSetter)(bool);
```

Comment:
using type alias (i.e., `using ...`) and `std::function` instead of function pointer.
https://en.cppreference.com/w/cpp/utility/functional/function

See Item 9 in http://shop.oreilly.com/product/0636920033707.do

Reply:
done in 7a731f9

Reply:
I decided not to use an alias because it is reasonably clear what the function does already and the alias won't get any use anywhere else in the code